### PR TITLE
Add `volumenameprefix` storageclass parameter in GlusterFS dynamic PV provisioning example section.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@sebgoa Can you please review/merge this PR ?

File: staging/persistent-volume-provisioning/README.md [hunk 1]
```diff
@@ -151,6 +151,12 @@ For available volume types and its administration options refer: ([Administratio
 
 For available volume options and its administration refer: ([Administration Guide](https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.2/html/administration_guide/chap-managing_red_hat_storage_volumes))
 
+* `volumenameprefix` : By default dynamic provisioned volumes has the naming schema of `vol_UUID` format. With this option present in storageclass, an admin can now prefix the desired volume name from storageclass. If `volumenameprefix` storageclass parameter has set, the dynamic volumes are created in below format where `_` is the field seperator/delimiter:
```
Comment (sebgoa) [inline staging/persistent-volume-provisioning/README.md]: dynmically provisioned.

Comment (sebgoa) [reply staging/persistent-volume-provisioning/README.md]: storageclass paramter is set

**Pull request merged**